src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    disabled: 'desabilitado',
+    disableAllVideo: 'Desabilitar todo vídeo',
+    dltDisabled: 'Pode ser deletado pelo criador da enquete',
+    documentShare: 'Compartilhar documento',
+    DocumentSharing: 'Compartilhamento de documento',
+    DocumentSharedbHeading: 'Painel de documento',
+    docUploadSuccess: 'Documento enviado com sucesso',
+    download: 'Baixar',
+    downloadFile: 'Baixar arquivo',
+    downloadsession: 'Favor aguardar enquanto a gravação é baixada',
+    drawArea: 'Área de desenho',
+    dropfilehere: 'Solte os arquivos aqui',
+    duplicateUploadMsg: 'Este arquivo já existe, por favor envie um novo arquivo.',
+    edit: 'Editar',
+    editorCode: 'Editor de código',
+    edittitle: 'Editar título',
+    editorRichDisable: 'Modo de escrita',
+    editorRichEnable: 'Apenas leitura',
+    educator: 'Educador',
+    EHTMLPresentUrl: 'Inserir URL da apresentação HTML5',
+    enable: 'Habilitar',
+    enablehistory: '"Histórico do navegador" precisa estar habilitado durante a sessão.',
+    enableAllVideo: 'Habilitar todo vídeo',
+    enteryouryoutubeurl: 'Inserir URL de vídeo online ou YouTube',
+    etDisabledA: 'Não pode editar, enquete já foi tentada',
+    etDisabledCr: 'Pode ser editada pelo criador da enquete',
+    ETime: 'Tempo decorrido',
+    FF2: 'Acelerar x2',
+    FF8: 'Acelerar x8',
+    filenotsave: 'Seu arquivo não pode ser salvo.',
+    filetsaveTS: 'A sessão está pronta para salvar.',
+    Finish: 'Finalizar',
+    fitToPage: 'Ajustar à página',
+    fitToScreen: 'Ajustar à tela',
+    Freehand: 'Mão livre',
+    general: 'Geral',
+    Greport: 'Relatório de nota',
+    highBandWidthSpeed: 'Sua banda larga é boa o suficiente.',
+    httpsmissing: 'Apenas origens seguras(https) são permitidas para compartilhamento de tela.',
+    ieBrowserIssue: 'Internet Explorer não é suportado, Congrea suporta completamente Chrome  e Firefox.',
+    ios7support: 'Para Apple, Sala Virtual suporta iOS 8 e versões superiores.',
+    iosAudEnable: 'Toque aqui para habilitar áudio',
+    InternalError: 'Por favor, certifique-se que a mesma webcam não está sendo usada <br /> em múltiplos browsers ou múltiplas aplicações. ',
+    invalidcmid: 'Id de módulo do curso inválido.',
+    invalidurl: 'URL inválida',
+    indvprogress: 'Tarefa atual',
+    InsertimageURL: 'Inserir URL da imagem',
+    JoinClassMsg: 'Juntar-se à Sala de Aula',
+    JoinSession: 'Juntar-se à sessão',
+    keymissing: 'A chave do seu software(LMS/CMS) está faltando',
+    Line: 'Linha',
+    lowBandWidthSpeed: 'Sua banda larga é muito baixa, nós vamos parar o vídeo. Você ainda poderá ouvir o áudio e ver a tela.',
+    maxCommonChat: 'Mostrar janela de bate-papo',
+    Max_rooms: 'Erro: Limite de máximo de salas alcançado',
+    Max_users: 'Erro: Limite de máximo de usuários alcançado',
+    media: 'Mídia',
+    mediumBandWidthSpeed: 'Sua banda larga é limitada, nós vamos reduzir a qualidade do vídeo de acordo com a banda larga disponível.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 135. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)